parameterspace/parameters/categorical.py

```diff
@@ -21,7 +21,7 @@ class CategoricalParameter(BaseParameter):
     def __init__(
         self,
         name: str,
-        values: Tuple[Any],
+        values: Tuple[Any, ...],
```

Comment:
How about switching to `Tuple`? from [typing docs](https://docs.python.org/3/library/typing.html#typing.Tuple)
"Tuple is equivalent to Tuple[Any, ...]"

Reply:
`Tuple[Any, ...]` seems more explicit (showing that we didn't just "forgot" the inner type). But I've no strong preference, do you?